tiledb/sm/serialization/array_schema_evolution.cc

```diff
@@ -153,7 +154,10 @@ Status array_schema_evolution_to_capnp(
   timestamp_builder.set(0, timestamp_range.first);
   timestamp_builder.set(1, timestamp_range.second);
 
-  // TODO: to add actual wire CurrentDomain (ch48253)
+  auto crd = array_schema_evolution->current_domain_to_expand();
+  auto current_domain_builder =
+      array_schema_evolution_builder->initCurrentDomainToExpand();
+  current_domain_to_capnp(crd, &current_domain_builder);
```

Comment:
What would happen if the schema evolution does not expand the current domain? Should we call `initCurrentDomainToExpand` only then?

Reply:
This is a great catch, thanks!